### PR TITLE
Update ETC.tsv

### PR DESCRIPTION
**Pull request by Sunaries**
- Updated.

### CODE REVIEW

**Comment by Sunaries** [comment]
Good to merge.

File: ETC.tsv [hunk 1]
```diff
@@ -14346,135 +14346,135 @@ ETC_20150918_014347
...
-ETC_20150918_014471	
-ETC_20150918_014472	
-ETC_20150918_014473	
-ETC_20150918_014474	
-ETC_20150918_014475	
-ETC_20150918_014476	
-ETC_20150918_014477	
-ETC_20150918_014478	
-ETC_20150918_014479	
+ETC_20150918_014379 Recipe - Escanciu Sword
```
Comment (ttgmichael) [inline ETC.tsv:14377]: Missing tab here?

File: ETC.tsv [hunk 2]
```diff
@@ -14288,12 +14288,12 @@ ETC_20150918_014289
 ETC_20150918_014290	
 ETC_20150918_014291	
 ETC_20150918_014292	
-ETC_20150918_014293	
-ETC_20150918_014294	
+ETC_20150918_014293 Siauliai Mission
```
Comment (ttgmichael) [inline ETC.tsv:14291]: here too?

Comment (Sunaries) [inline ETC.tsv:14291]: Strange, because it's just fine in Notepad++ and in the Github editor.

File: ETC.tsv (hunk 1 above)
Comment (Sunaries) [inline ETC.tsv:14377]: Strange, because it's just fine in Notepad++ and in the Github editor.

**Comment by imcgames** [comment]
Thank you :smile:

**Pull request merged**